zilencer/management/commands/populate_db.py

```diff
@@ -834,6 +835,27 @@ def assign_time_zone_by_delivery_email(delivery_email: str, new_time_zone: str)
                     mit_realm, testsuite_mit_users, tos_version=settings.TERMS_OF_SERVICE_VERSION
                 )
 
+                mit_user = get_user_by_delivery_email("sipbtest@mit.edu", mit_realm)
+                mit_signup_stream = Stream.objects.get(
+                    name=Realm.INITIAL_PRIVATE_STREAM_NAME, realm=mit_realm
+                )
+                mit_signup_recipient = Recipient.objects.get(
+                    type=Recipient.STREAM, type_id=mit_signup_stream.id
+                )
+                mit_signup_sub = Subscription(
+                    recipient=mit_signup_recipient,
+                    user_profile=mit_user,
+                    is_user_active=mit_user.is_active,
+                )
+                mit_log = RealmAuditLog(
+                    realm=mit_realm,
+                    modified_user=mit_user,
+                    modified_stream=mit_signup_stream,
+                    event_last_message_id=0,
+                    event_type=RealmAuditLog.SUBSCRIPTION_CREATED,
+                    event_time=event_time,
+                )
```

Comment:
I don't like add hand-creation like this in `populate_db` because it can result in invalid states if we forget to do things (like `RealmAuditLog`); can we just call `bulk_add_subscriptions`? 

Generally we want to be moving in the direction of this setup logic being shared with production code.

Reply:
I did not call `bulk_add_subscriptions` since I noticed we do not do it in other places and thought it was to avoid additional events related calls.

Reply:
It's a bit cheaper to do that, but that decision was years ago, before we had things like RealmAuditLog that would be large blocks of copied code.

I think our goal should generally be to use `actions` functions inside `populate_db` except where there's a clear performance or other benefit to doing otherwise.

Reply:
(So we should change this, and think about changing other places too where convenient).